extensions/azurePublishNew/src/recoilModel/atoms/importConfigurationState.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { atom } from 'recoil';
+
+import { ImportConfiguration } from '../../types';
+
+export const importConfigurationState = atom<ImportConfiguration>({
+  key: 'import_configuration',
+  default: { isValidConfiguration: false, config: '' },
```

Comment:
config default value should be ```{}```
instead of a boolean, keep track of JSON.parse error so you can show it in the UX instead of a "invalid JSON" message.